webpack/assets/javascripts/react_app/components/common/table/reducers/createTableReducer.js

```diff
@@ -11,25 +11,42 @@ const initState = Immutable({
 });
 
 const createTableReducer = tableID => (state = initState, action) => {
-  const { REQUEST, FAILURE, SUCCESS } = createTableActionTypes(tableID);
-
-  switch (action.type) {
+  const { REQUEST, FAILURE, SUCCESS, SET_PAGINATION } = createTableActionTypes(
```

Comment:
Looking at changes in [Table](https://github.com/theforeman/foreman/pull/7061/files#diff-ca9c42cdb7200e236a4ba7a5da4470ceR41), are we ok with having SET_PAGINATION for tables that are not paginated?

Reply:
It will just be not used, I don't see the harm in that